setup.py

```diff
@@ -17,9 +17,8 @@
     "eve>=1.1.2,<=2.1.0",
     "eve-elastic>=7.4.0,<7.5.0",
     "elasticsearch[async]<7.18",  # we are using oss version on test server
-    "flask>=1.1,<1.2",
+    "flask>=3.0",
```

Comment:
We will need to include the `async` extra for Flask.
i.e. `flask[async]>=3.0`

Reply:
done